packages/twenty-server/src/engine/core-modules/auth/services/google-apis.service.ts

```diff
@@ -160,6 +160,10 @@ export class GoogleAPIsService {
         `UPDATE ${dataSourceMetadata.schema}."connectedAccount" SET "accessToken" = $1, "refreshToken" = $2, "authFailedAt" = NULL WHERE "id" = $3`,
         [accessToken, refreshToken, connectedAccountId],
       );
+      await manager.query(
```

Comment:
I don't understand why we have this reset instruction in the updateConnectedAccount function